auth/cert/loader.go

```diff
@@ -30,6 +31,18 @@ func loadCertificates(caCertPool *x509.CertPool, certificatesDirPath string) err
 	return nil
 }
 
+func LoadCertificate(clientCertPath, clientCertKeyPath string) (certificate tls.Certificate, err error) {
+	certificate, err = tls.LoadX509KeyPair(clientCertPath, clientCertKeyPath)
+	if err != nil {
+		if clientCertKeyPath == "" {
+			err = errorutils.CheckErrorf("failed using the certificate located at %s. Reason: %s. Hint: No key for the certificate was provided.", clientCertPath, err.Error())
```

Comment:
failed using the certificate located at %s. Reason: %s. Hint: No key for the certificate was provided.
-->
failed using the certificate located at %s. Reason: %s. Hint: A certificate key was not provided. Make sure that the certificate doesn't require a key